### PR TITLE
change to x-www-form-urlencoded

### PR DESCRIPTION
**Pull request by li-sherry**
https://api.slack-gov.com/methods/users.lookupByEmail requires x-www-form-urlencoded

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @friggframework/api-module-ironclad@0.0.24-canary.107.75550ee.0
  npm install @friggframework/api-module-slack@0.1.16-canary.107.75550ee.0
  # or 
  yarn add @friggframework/api-module-ironclad@0.0.24-canary.107.75550ee.0
  yarn add @friggframework/api-module-slack@0.1.16-canary.107.75550ee.0
  ```
</details>

### CODE REVIEW

File: api-module-library/slack/api.js [hunk 1]
```diff
@@ -81,7 +81,8 @@ class Api extends OAuth2Requester {
             return this._request(url, options, i + 1);
         } else if (
             parsedResponse.error === 'invalid_auth' ||
-            parsedResponse.error === 'auth_expired'
+            parsedResponse.error === 'auth_expired' ||
+            parsedResponse.error === 'token_expired'
```
Comment (li-sherry) [inline api-module-library/slack/api.js:85]: not sure if auth_expired exists for slack?

Comment (seanspeaks) [reply api-module-library/slack/api.js:85]: I think it was returned in a test at one point? Not in the docs anywhere though, agreed.

File: api-module-library/slack/api.js [hunk 2]
```diff
@@ -144,7 +145,7 @@ class Api extends OAuth2Requester {
             url: this.baseUrl + this.URLs.lookupUserByEmail,
             body,
             headers: {
-                'Content-Type': 'application/json',
+                'Content-Type': 'application/x-www-form-urlencoded',
```
Comment (seanspeaks) [inline api-module-library/slack/api.js:148]: What an odd thing to require by Slack. But, good catch!

**Review by seanspeaks — APPROVED** [review]
At some point we'll make these api tests easy to write; for now, all looks good from here.

And you're right, this should capture/trigger the auth refresh flow

File: api-module-library/slack/api.js (hunk 2 above)
Comment (li-sherry) [reply api-module-library/slack/api.js:148]: in postman it works even when content-type is json, but the docs say x-www-form-urlencoded

**Comment by seanspeaks** [comment]
:rocket: PR was released in [`@friggframework/api-module-slack@0.1.18`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-slack%400.1.18) :rocket:

**Comment by seanspeaks** [comment]
:rocket: PR was released in [`@friggframework/api-module-ironclad@0.0.26`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-ironclad%400.0.26) :rocket:

**Pull request merged**